src/formatter/helpers/issue_helpers.js

```diff
@@ -93,6 +93,18 @@ function formatStep({
   }
   text += '\n'
 
+  if (Array.isArray(testStep.attachments)) {
+    testStep.attachments
+      .filter(({ media }) => media.type === 'text/plain')
+      .forEach(({ data }) => {
+        text +=
+          indentString(
+            colorFns[Status.UNDEFINED](figures.info + ' ' + data),
+            4
+          ) + '\n'
+      })
+  }
```

Comment:
Thoughts on attachments being added after the step argument so it appears after the docstring / data table and not in between?

Reply:
It seems to be append always between step line and docstring/data.

Reply:
If this was block was placed after the next block (that includes buildStepArgumentIterator) I believe it would appear after the step argument. Please add a test for this as well